README.md

```diff
@@ -10,62 +10,86 @@ Please find project details at [Wiki](https://github.com/Samsung/TizenRT/wiki) e
 
 > [Quick Start](#quick-start)  
 > [Supported Board / Emulator](#supported-board--emulator)  
-> [Configuration Sets](#configuration-sets)
 
 ## Quick Start
-### Getting the toolchain
 
-Install the OS specific toolchain. Supported OS Type's are "linux" and "mac".  
-Get the build in binaries and libraries, [gcc-arm-none-eabi-6-2017-q1-update-*OS Type*.tar.bz2](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads/6-2017-q1-update)  
-Untar the gcc-arm-none-eabi-6-2017-q1-update-*OS Type*.tar.bz2 and export the path like
+TizenRT provides the easiest way to build with the use of [Docker](https://www.docker.com/).  
+There is no need to install the required libraries and toolchains since the provided Docker container already includes everything required for TizenRT development.  
+However, if your development systems are not eligible for running the Docker container, all libraries and toolchains should be manually installed.  
+Please refer to [Manual Setup Build Environment](docs/HowToSetEnv.md).
 
-```bash
-tar xvjf gcc-arm-none-eabi-6-2017-q1-update-[OS Type].tar.bz2
-export PATH=<Your Toolchain PATH>:$PATH
-```
-Be aware that recommanded toolchain is fully working on 64bits machine.
+For more infomation of libraries in the TizenRT Docker Image, see https://hub.Docker.com/r/tizenrt/tizenrt/.
+
+### 1. Install Docker
 
-NOTE: See [Additional Requirements](docs/InstallAdReq.md) for gRPC, ROMFS, OCF and IoT.js.
+To install OS specific Docker engines, see https://docs.docker.com/install/linux/docker-ce/ubuntu/.  
+If you already have a Docker engine, please skip this step.
 
-### Getting the sources
+### 2. Getting TizenRT source code
 
 ```bash
 git clone https://github.com/Samsung/TizenRT.git
 cd TizenRT
 TIZENRT_BASEDIR="$PWD"
 ```
+**Note**: To contribute in this community, fork this project first into own github repository and do git cloning from your repository. 
+> git clone https://github.com/your_github_id/TizenRT.git
+
+
+### 3. Build
 
-### How to Build
+#### 3.1 Configuration (*$TIZENRT_BASEDIR/os/tools*)
 
-Configure the build from *$TIZENRT_BASEDIR/os/tools* directory
 ```bash
 cd os/tools
```

Comment:
Let's modifiy here as shown below:
```
cd os
./tools/configure.sh <board>/<configuration_set>
```
And let's remove all of ($TIZENRT_BASEDIR/xxxx) at end of sub-title like 3.1, 3.2.

Reply:
We can keep same location to execute all of command.